### PR TITLE
Changes to font (aesthetics) and added r project

### PR DESCRIPTION
**Pull request by RRMaximiliano**
There are some other changes in the data processing and viz slides. This should be the most updated version of all the slides as of now.

### CODE REVIEW

File: Presentations/03-data-visualization.Rmd [hunk 1]
```diff
@@ -129,22 +143,15 @@ Similarly to previous sessions, you can find some references at the end of this
...
-# Replace with where your data is
-
-FolderPath <- file.path("YOUR/FOLDER/PATH")
-
-dataWorkFolder    <- file.path(projectFolder,"DataWork")
-Data              <- file.path(dataWorkFolder,"DataSets")
-finalData         <- file.path(Data,"Final")
-
-whr_panel <- readRDS(file.path(finalData, "whr_panel.RDS"))
+here::here()
```
Comment (luisesanmartin) [inline Presentations/03-data-visualization.Rmd:149]: this line might be missing the command to load the csv file

Comment (luizaandrade) [reply Presentations/03-data-visualization.Rmd:149]: This was fixed in #63

File: Presentations/03-data-visualization.Rmd [hunk 2]
```diff
@@ -129,22 +143,15 @@ Similarly to previous sessions, you can find some references at the end of this
...
 
 ```
 
 ```{r echo = FALSE, include = FALSE}
-whr_panel <- readRDS(file.path(finalData, "whr_panel.RDS"))
+whr_panel <- readRDS(here::here("DataWork", "DataSets", "Final", "whr_panel.Rds"))
```
Comment (luizaandrade) [inline Presentations/03-data-visualization.Rmd:154]: I removed the `here::` part in #63 to make this consistent with how we use other packages. Was there a particular reason why you included it, @RRMaximiliano?

File: Presentations/03-data-visualization.Rmd [hunk 3]
```diff
@@ -1245,4 +1206,60 @@ https://www.coursera.org/learn/exploratory-data-analysis
...
+
+```{r, message=FALSE, warning=FALSE, out.width = "70%", out.height="70%", out.extra = 'style="display:block; margin:auto;"'}
+# Load package
+library(plotly)
+
+# Use ggplotly to create an interactive plot
+ggplotly(fig) %>% 
+  layout(legend = list(orientation = "h", x = 0.4, y = -0.2))
+
+```
```
Comment (luizaandrade) [inline Presentations/03-data-visualization.Rmd:1265]: This was not compiling for me after the changes in #63. Can you check that it works for you?

**Review by luizaandrade — APPROVED** [review]

File: Presentations/02-data-processing.Rmd [hunk 4]
```diff
@@ -140,29 +156,115 @@ Takeaway: long format > wide format
...
+---
+
+# Introduction: RStudio Projects
+
+### RStudio projects give you a solid workflow: 
+
+- Uses relative paths. 
+- Keep data files there. 
+- You will have a history of the functions you have used.
+- It could load your the last state of your environment.
```
[the remark this answers is not in the document]
Comment (RRMaximiliano) [reply Presentations/02-data-processing.Rmd:189]: Fixed the typo here.

File: Presentations/02-data-processing.Rmd [hunk 5]
```diff
@@ -27,9 +30,20 @@ opts_chunk$set(
   cache=T#, echo=F, warning=F, message=F
   )
 library(hrbrthemes)
+library(here)
```
[the remark this answers is not in the document]
Comment (RRMaximiliano) [reply Presentations/02-data-processing.Rmd]: Added tidyverse here

File: Presentations/02-data-processing.Rmd [hunk 6]
```diff
@@ -140,29 +156,115 @@ Takeaway: long format > wide format
...
 
 # Introduction
 
 ### Before we start, let's make sure we are all set: 
 
 1. Start a fresh RStudio session.
-2. Load the tidyverse package.
-3. Set your file paths.
-
+2. Open your RStudio project.
```
[the remark this answers is not in the document]
Comment (RRMaximiliano) [reply Presentations/02-data-processing.Rmd:239]: Fixed a bit the wording.

File: Presentations/02-data-processing.Rmd [hunk 7]
```diff
@@ -240,6 +335,17 @@ The `clean_names()` function helps us big time when our variables names are pret
...
 > **Pro tip**: Use the `clean_names()` function in a pipe after you load a dataset as we did in the last slide.
 
+If we wanto to rename our variable manually, we could use: 
+
+```{r manual_names, eval=FALSE}
+
+whr15 <- whr15 %>% 
+  rename(
+    var_newname = var_oldname
+  )
```
[the remark this answers is not in the document]
Comment (RRMaximiliano) [reply Presentations/02-data-processing.Rmd:318]: Added pipe explanation. Forgot to add it before

File: Presentations/02-data-processing.Rmd [hunk 8]
```diff
@@ -963,10 +1004,100 @@ The R equivalent of a `.dta` file is a `.Rds` file. It can be saved and loaded u
...
+```{r select}
+whr15 %>% 
+  select(region, country, happiness_rank)
+```
+
+]
+
+.panel[.panel-name[Combining functions]
+
+`Select`: allows you to specific columns.
```
[the remark this answers is not in the document]
Comment (RRMaximiliano) [reply Presentations/02-data-processing.Rmd:1047]: Fixed

File: Presentations/02-data-processing.Rmd [hunk 9]
```diff
@@ -939,7 +980,7 @@ The function takes the following structure:
 # Save the whr data set
 
 write.csv(whr_panel, 
-          file.path(finalData,"whr_panel.csv"), 
+          here::here("DataWork", "DataSets", "Final", "whr_panel.csv"),
```
[the remark this answers is not in the document]
Comment (RRMaximiliano) [reply Presentations/02-data-processing.Rmd]: True. Forgot to remove those.

File: Presentations/02-data-processing.Rmd [hunk 10]
```diff
@@ -662,9 +706,9 @@ Similar in nature but:
 
 # Creating variables by groups
 
-Let's imagine now that we want to create a variable at the region level -- recal `bys gen` in Stata.In R, we can use `group_by()` before we mutate. So for this example, we going to combine the chain the following functions. 
+Let's imagine now that we want to create a variable at the region level (recall `bys gen` in Stata).In R, we can use `group_by()` before we mutate. So for this example, we going to pipe the following functions.
```
[the remark this answers is not in the document]
Comment (RRMaximiliano) [reply Presentations/02-data-processing.Rmd:687]: Fixed

File: Presentations/02-data-processing.Rmd [hunk 11]
```diff
@@ -582,7 +641,7 @@ name: variables
 
 ### In the tidyverse, we refer to creating variables as mutating
 
-So, instead of **gen**erate, we use `mutate()`. Let's say we want to have interactions:
+So, we use the `mutate()` function. Let's say we want to have interactions:
```
[the remark this answers is not in the document]
Comment (RRMaximiliano) [reply Presentations/02-data-processing.Rmd:622]: I removed arrange from this slide

File: Presentations/03-data-visualization.Rmd (hunk 2 above)
Comment (RRMaximiliano) [reply Presentations/03-data-visualization.Rmd]: There is no conflict with other package, I am just used to use here as here::

**Review by luizaandrade — APPROVED** [review]

File: Presentations/02-data-processing.Rmd [hunk 12]
```diff
@@ -140,29 +161,115 @@ Takeaway: long format > wide format
...
 # Introduction
 
 ### Before we start, let's make sure we are all set: 
 
 1. Start a fresh RStudio session.
-2. Load the tidyverse package.
-3. Set your file paths.
-
+2. Open your RStudio project.
+3. Load the tidyverse package.
```
[the remark this answers is not in the document]
Comment (RRMaximiliano) [reply Presentations/02-data-processing.Rmd:229]: Updated in a first that we have the 3 packages loaded at the beg of the session. And I explained first how to install and how to load in the same slide.

File: Presentations/02-data-processing.Rmd [hunk 13]
```diff
@@ -450,18 +576,17 @@ whr15 %>%
...
 whr15 %>% 
   filter(grepl("America", region)) %>% 
   head(2)
-
 ```
 
 --
 
-> Notice that I have used `head()` to show just the first 2 observations of the subset. If you want to save this subset you can assign it to an objet. For example `whr15_east <- + the code above`. Remember that the arrow sign (<-) is a mix of < and -. The font we are using in this presentation is called Fira Code, and it combines the two characters (a ligature). 
+> Notice that I have used `head()` to show just the first 2 observations of the subset. If you want to save this subset you can assign it to an objet. For example `whr15_east <- + the code above`.
```
[the remark this answers is not in the document]
Comment (RRMaximiliano) [reply Presentations/02-data-processing.Rmd:584]: I see your point. I love regex but you are right. I deleted the whole slide here.

**Comment by luisesanmartin** [comment]
@RRMaximiliano I think we can delete this branch now 👍

**Pull request merged**